Modules/zlibmodule.c

```diff
@@ -1372,6 +1834,11 @@ static PyMethodDef Decomp_methods[] =
     {NULL, NULL}
 };
 
+static PyMethodDef ZlibDecompressor_methods[] = {
+    ZLIB_ZLIBDECOMPRESSOR_DECOMPRESS_METHODDEF
+    {NULL}
```

Comment:
This might make a difference.
```suggestion
    {NULL, NULL}
```

Reply:
Thanks. This wasn't it, but thanks to this suggestion I found that I had forgotten the NULL terminator of ZlibDecompressor_Members.